packages/compass-workspaces/src/stores/workspaces.ts

```diff
@@ -466,11 +468,12 @@ export const getActiveTab = (state: WorkspacesState): WorkspaceTab | null => {
 };
 
 export type OpenWorkspaceOptions =
-  | Pick<Workspace<'My Queries'>, 'type'>
-  | Pick<Workspace<'Databases'>, 'type'>
-  | Pick<Workspace<'Performance'>, 'type'>
-  | Pick<Workspace<'Collections'>, 'type' | 'namespace'>
-  | (Pick<Workspace<'Collection'>, 'type' | 'namespace'> &
+  | Pick<Workspace<'Welcome'>, 'type'>
+  | Pick<Workspace<'My Queries'>, 'type' | 'connectionInfoId'>
```

Comment:
From the designs, "My Queries" is the only one that won't be per connection

Reply:
(Noticed because I've been adding it to the top level just now 😄 )

Reply:
It is now not per connection. Thank you for pointing this out. 🙌